File: 1 - Sequenciais/sequenciais7.c
//Faça um programa que calcule a área de um quadrado, em seguida mostre o dobro desta área para o usuário.

#include <stdio.h>

int main()
{
  float lado;
  printf("Digite o valor do lado do quadrado: \n");
  scanf("%f", &lado);
  printf("O dobro da area do quadrado eh de: %.2f", (lado * lado) * 2);
}

File: 1 - Sequenciais/Java/sequenciais7.java
//Faça um programa que calcule a área de um quadrado, em seguida mostre o dobro desta área para o usuário.

import java.util.Scanner;

public class sequenciais7 {

  public static void main(String[] args) {
    System.out.println("Digite o valor da lateral do quadrado: ");
    float lado = new Scanner(System.in).nextFloat();
    System.out.println("O dobro da area do quadrado eh de: " + ((lado * lado) * 2) );
  }

}
